src/Tests/Microsoft.NET.Restore.Tests/GivenThatWeWantToRestoreDotNetCliToolReference.cs

```diff
@@ -0,0 +1,101 @@
+// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.IO;
+using System.Linq;
+using System.Xml.Linq;
+using FluentAssertions;
+using Microsoft.NET.TestFramework;
+using Microsoft.NET.TestFramework.Assertions;
+using Microsoft.NET.TestFramework.Commands;
+using Microsoft.NET.TestFramework.ProjectConstruction;
+using NuGet.Common;
+using NuGet.Frameworks;
+using NuGet.ProjectModel;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Microsoft.NET.Restore.Tests
+{
+    public class GivenThatWeWantToRestoreDotNetCliToolReference : SdkTest
+    {
+        private const string ProjectToolVersion = "1.0.0";
+        private const string ExpectedProjectToolRestoreTargetFrameworkMoniker = "netcoreapp2.2";
+
+        public GivenThatWeWantToRestoreDotNetCliToolReference(ITestOutputHelper log) : base(log)
+        {
+        }
+
+        [Fact]
+        public void It_can_restore_with_netcoreapp2_2()
```

Comment:
@dsplaisted I find previous test restoring aspnet scaffolding is too slow and download too much, and may cause fullframework leg time out. So I changed it to restore a sample project tool and only asset restore's TFM